### PR TITLE
feat: Add graph_Cooccurence and samples

### PR DESCRIPTION
**Pull request by jbrundage**
Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [x] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by jbrundage** [comment]
@GordonSmith this is ready for rereview

File: packages/graph/src/Cooccurence.ts [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+            ;
+        colEnter
+            .each(function (d, i) {
+                const row = d3Select(this);
+                row.append("text");
+                row.append("rect");
+            })
+            ;
+        colEnter
+            .merge(colSelection)
```
[the remark this answers is not in the document]
Comment (GordonSmith) [reply packages/graph/src/Cooccurence.ts]: @jbrundage bump

File: packages/graph/src/Cooccurence.ts [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+                    .style("font-family", context.fontFamily())
+                    .text(d)
+                    ;
+                row.select("rect")
+                    .attr("height", context.height())
+                    .attr("width", 1)
+                    .attr("fill", "white")
+                    ;
+            });
+
```
[the remark this answers is not in the document]
Comment (GordonSmith) [reply packages/graph/src/Cooccurence.ts]: @jbrundage bump `colSelection.exit().remove()`

File: packages/graph/src/Cooccurence.ts [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+type EdgeRowType = [string, string, number];
+
+export class Cooccurence extends SVGWidget {
+
+    _g;
+    _backgroundElement;
+    _drawStartPos;
+    private _edges: EdgeRowType[] = [];
+    _palette;
```
Comment (GordonSmith) [inline packages/graph/src/Cooccurence.ts]: They all need types + protected / private

**Comment by jbrundage** [comment]
@GordonSmith this is ready for rereview

**Comment by GordonSmith** [comment]
Needs rebase...

**Pull request closed without merging**